swagger2.0/oic.r.notificationselector-pushproxy.swagger.json

```diff
@@ -0,0 +1,334 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Notification Selector",
+    "version": "2019-03-25",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "Copyright 2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/NotificationSelectorPushproxyResURI" : {
+      "get": {
+        "description": "Resource that defines the selector for Push Notifications",
+        "parameters": [
+          {"$ref": "#/parameters/interface-rw"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "",
+            "x-example": {
+              "phref": "/myAirquality",
```

Comment:
please add rt in the get examples everywhere

Reply:
I'm little bit confused here.
According to the core spec, difference b/w `oic.if.baseline` and other interface such as `oic.if.rw` is that baseline shall include common Properties. 
Current iotiviti-lite returns Properties except for common Properties only against RETRIEVE through `oic.if.rw` interface.
But there is no specific text limiting non-common Properties when `oic.if.rw` interface is applied in the spec. 

Which is the correct operation ?

Reply:
ok leave it like this.